libnavigation-core/src/test/java/com/mapbox/navigation/core/MainCoroutineRule.kt

```diff
@@ -1,4 +1,4 @@
-package com.mapbox.navigation
+package com.mapbox.navigation.core
```

Comment:
I believe this should live in the testing module and be consumed from the rest of the modules. Also IMO we should have a generic `libtesting` module instead of `libtesting-ui` as we do right now. What do you think?

Reply:
Agreed - https://github.com/mapbox/mapbox-navigation-android/issues/2411.